### PR TITLE
[Azure AD roles] allowedToReadBitlockerKeysForOwnedDevice property

### PR DESCRIPTION
**Pull request by rolyon**

### CODE REVIEW

**Review by loradule — COMMENTED** [review]
Could we please change the value of the new property to be true instead of Boolean, to match the others?

**Review by loradule — APPROVED** [review]

File: api-reference/beta/toc.yml [hunk 1]
```diff
@@ -12697,6 +12697,10 @@ items:
               href: api/identityapiconnector-uploadclientcertificate.md
             - name: Delete
               href: api/identityapiconnector-delete.md
+        - name: User
+          items:
+          - name: Default user role permissions
```
Comment (Lauragra) [inline api-reference/beta/toc.yml:12702]: FYI, we don't expose complex types in the TOC; I recommend removing this entry.

Comment (FaithOmbongi) [reply api-reference/beta/toc.yml:12702]: +1

Comment (rolyon) [reply api-reference/beta/toc.yml]: Sure, I removed TOC entry

**Pull request merged**